src/frontend/Href.elm

```diff
@@ -28,6 +29,11 @@ toModule author project version moduleName maybeValue =
   Url.custom Url.Absolute [ "packages", author, project, vsnToString version, String.replace "." "-" moduleName ] [] maybeValue
 
 
+toAbout : String -> String -> Maybe V.Version -> String
```

Comment:
I think I wouldn't add this. Right now all the header segments for packages correspond to parts of `elm.json` files: package names, versions, and module names.

I'm inclined to keep the header the same for `README.md` and for `elm.json` right now, and to think more about what we want the header to convey before. Maybe the answer is that the `/help` pages should also not have stuff in the header.